builder/tealium-swift.xcodeproj/xcshareddata/xcschemes/TealiumAttribution.xcscheme

```diff
@@ -1,95 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hi @craigrouse, due to this file being deleted, the `TealiumAttribution` scheme is not a shared scheme anymore, which means Carthage is not able to build it. Could you please share the `TealiumAttribution` scheme again and create a new release?

Reply:
Hi @ffittschen, sorry for the inconvenience here! It looks like the .xcscheme didn't get pushed to GH for some reason. We have this queued up for the next release and will keep you posted. Thanks!

Reply:
@ffittschen, we just released the fix for this in version [2.2.1](https://github.com/Tealium/tealium-swift/releases/tag/2.2.1) 🙂 Let us know how it goes!